pkg/types/hashedrekord/hashedrekord_test.go

```diff
@@ -150,10 +152,36 @@ func TestToLogEntry(t *testing.T) {
 				Digest: hexDecodeOrDie(t, hexEncodedDigest),
 			},
 		},
+		{
+			name: "mismatched key algorithm",
+			hashedrekord: &pb.HashedRekordRequestV0_0_2{
+				Signature: &pb.Signature{
+					Content: b64DecodeOrDie(t, b64EncodedSignature),
+					Verifier: &pb.Verifier{
+						Verifier: &pb.Verifier_PublicKey{
+							PublicKey: &pb.PublicKey{
+								RawBytes: []byte(publicKey),
+							},
+						},
```

Comment:
Can this test include `key_details` explicitly, since it missing is technically a malformed request?